ts/features/bonus/bpd/api/patchedTypes.ts

```diff
@@ -24,3 +24,21 @@ export const PatchedCitizenResource = t.intersection(
 );
 
 export type PatchedCitizenResource = t.TypeOf<typeof PatchedCitizenResource>;
+
+/**
+ * patched version of CitizenV2Resource
+ * - technicalAccount must be optional
+ */
+// required attributes
+const PatchedCitizenV2ResourceO = t.interface({
```

Comment:
If it must be optional you should `partial` codec
```suggestion
const PatchedCitizenV2ResourceO = t.partial({
```

Reply:
Sure, cut and paste error.
Fixed in [517564f](https://github.com/pagopa/io-app/pull/2969/commits/517564fef97e674558a83e418c139af1c2e1d759)